host/switch.go

```diff
@@ -0,0 +1,189 @@
+package main
```

Comment:
This file appears to be Windows-only? If yes, please call it `switch_windows.go` (that's usually more readable than build tags, though that's missing here too).